packages/shared/lib/clients/sync.client.ts

```diff
@@ -46,6 +46,10 @@ const OVERLAP_POLICY: ScheduleOverlapPolicy = ScheduleOverlapPolicy.BUFFER_ONE;
 
 const namespace = process.env['TEMPORAL_NAMESPACE'] || 'default';
 
+export interface RecordsServiceInterface {
+    deleteRecordsBySyncId({ syncId }: { syncId: string }): Promise<{ totalDeletedRecords: number }>;
+}
```

Comment:
Sync it's a type it could live inside records too

Reply:
`shared` doesn't depend on `records` :D

Reply:
as a devDeps yes so it would have worked for types :D